### DISCUSSION TITLE
Question 99: When to use GLOBUS v Dropbox?

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#909] [Q&A]
Description: What are the recommendations for when to use globus v dropbox?  WS has an alternative system that he uses to share with folks outside of the museum. KW to follow up

**Comment by rondlg** [comment]
Response: IT to write up guidelines and share with others. The Museum has a paid account and FM and non-FM users do not need a paid account to use it and be able to share back and forth.

**Comment by magpiedin** [comment]
Note: Finished projects/media/datasets should be properly documented, published and archived following 

# Globus ([globus.org](http://globus.org/)) - data/file transfer service

## When to use it:

- After requesting imaging_scratch space, if/when FMNH staff need to transfer large files/datasets (e.g. CT scans) to or from outside institutions/people.

You will be asked to set up an event in EMu.
- https://github.com/fieldmuseum/EMu-Documentation/discussions/346

## How to use it:

1. [Request imaging_scratch space](https://github.com/fieldmuseum/EMu-Documentation/discussions/346)
3. [Install, Configure, and Uninstall Globus Connect Personal for Windows](https://docs.globus.org/how-to/globus-connect-personal-windows)


# Dropbox 
- Consider this very working space for sharing smaller temporary files with outside collaborators.  


_Related discussion: [EMu Hour - 2023 Sep 26](https://github.com/fieldmuseum/EMu-Documentation/discussions/906)_
